snakedenfitness/fitness/views.py

```diff
@@ -44,11 +47,30 @@ def workout_form(request):
 
     return render(request, 'fitness/workout_form.html', {'form': form})
 
-
 def update_profile(request, user_id):
     user = User.objects.get(pk=user_id)
     user.profile.bio = 'Lorem ipsum dolor sit amet, consectetur adipisicing elit...'
     user.save()
 
 def request_trainer(request):
-    return render(request, 'fitness/request_trainer.html', {})
\ No newline at end of file
+    return render(request, 'fitness/request_trainer.html', {})
+
+def delete_workout(request, id):
+    workout = Workout.objects.get(pk=id)
+    workout.delete()
+    return redirect('fitness_home')
+    return render(request, 'fitness/delete_workout.html', {})
+
+def edit_workout(request):
```

Comment:
need to delete this function because of the duplicate function below

Reply:
deleted